smsapi/Api/Action/AbstractAction.php

```diff
@@ -2,24 +2,33 @@
 
 namespace SMSApi\Api\Action;
 
+use Exception;
+use SMSApi\Api\Action\Contacts\ContactsAction;
```

Comment:
I agree with you, but this nasty trick is necessary. It will be removed in near future, I promise ;)